core/services/workflows/engine.go

```diff
@@ -919,6 +920,20 @@ func (e *Engine) executeStep(ctx context.Context, lggr logger.Logger, msg stepRe
 	if err != nil {
 		return nil, nil, err
 	}
+	stepTimeoutDuration := e.stepTimeoutDuration
+	if timeoutOverride, ok := config.Underlying[reservedFieldNameStepTimeout]; ok {
```

Comment:
is this config set per capability? Checking to make sure its not set at the workflow level

Reply:
Yes, see example in the test file